runtime/onert/core/include/backend/basic/Tensor.h

```diff
@@ -41,8 +41,8 @@ class Tensor : public IPortableTensor
 
 public:
   Tensor(const ir::OperandInfo &info, DynamicMemoryManager *dynamic_mem_mgr)
-    : IPortableTensor(info), _layout(ir::Layout::NHWC), _buffer(nullptr), _num_references(0),
-      _dynamic_mem_mgr(dynamic_mem_mgr), _allocator(nullptr)
+    : IPortableTensor(info), _layout(ir::Layout::NHWC), _buffer(nullptr), _size(info.total_size()),
```

Comment:
(optional)
```suggestion
    : IPortableTensor(info), _layout(ir::Layout::NHWC), _buffer(nullptr), _size(0),
```

It doesn't matter what the initialized value of `_size` is. So it can be zero.

Reply:
> It doesn't matter what the initialized value of _size is. So it can be zero.

There is no code to set `_size` anywhere. Is it OK to initialize `_size` as zero?

Reply:
>It doesn't matter what the initialized value of _size is. So it can be zero.

Static memory allocator does not set `_size`. Without this setting, runtime use dynamic allocator on first shape change when size is smaller.